nyaa/api_torznab.py

```diff
@@ -101,71 +112,113 @@ def torznab_caps():
     return render_torznab_caps()
 
 
-def torznab_search():
-    page_size = app.config['RESULTS_PER_PAGE']
+def get_categories_filter(cat):
+    categories = []
+    if cat:
+        for item in cat.split(','):
+            # Parse 1_1 format for compatibility
```

Comment:
It's your implementation, so your choice, but this is essentially set arithmetic.  If you have a set with no items in it, that's the Null set.   You take a set with items, and remove all the items from it, what do you have left? Answer: The Null set.  For me that's the principle of least astonishment.